enhancements/ingress/add-headers-to-haproxyconfig-via-ingress-controller-or-route-api.md

```diff
@@ -0,0 +1,295 @@
+---
+title: add-headers-to-haproxyconfig-via-ingress-controller-or-route-api
+authors:
+  - "@miheer"
+reviewers:
+  - "@frobware"
+  - "@gcs278"
+  - "@Miciah"
+approvers:
+  - "@frobware"
+  - "@gcs278"
+  - "@Miciah"
+api-approvers:
+  - "@frobware"
+  - "@gcs278"
+  - "@Miciah"
+creation-date: 2020-06-10
+last-updated: 2020-06-10
+tracking-link:
+  - "https://issues.redhat.com/browse/NE-982"
+status: implemented
```

Comment:
This line needs to be updated or deleted.

Reply:
done. Added TODO